AzureCalling/External/Calling/TableViewDataSource.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class TableViewDataSource: NSObject, UITableViewDataSource {
```

Comment:
I think it would be simpler if we kept this as part of the `AudioDeviceSelectionViewController`.

Reply:
The idea behind was also to be generic enough to be reused for the participant list